Source/WebGPU/WGSL/EntryPointRewriter.cpp

```diff
@@ -131,7 +131,7 @@ void EntryPointRewriter::collectParameters()
     while (m_function.parameters().size()) {
         auto parameter = m_shaderModule.takeLast(m_function.parameters());
         Vector<String> path;
-        visit(path, MemberOrParameter { parameter->name(), parameter->typeName(), WTFMove(parameter->attributes()) });
+        visit(path, MemberOrParameter { parameter->name(), parameter->typeName(), parameter->attributes() });
```

Comment:
Is this related?

Reply:
It's somewhat related, and I thought it was better to include it here since it's a single line. This is the last place that modified the AST in a way that wasn't reversible (by moving the attributes out of the parameter), this caused this patch to not work. Since we compile the vertex and fragment shaders separately, the second compilation would fail due to the missing attributes.